docs/website/sidebars.json

```diff
@@ -34,19 +34,21 @@
       "concepts/i18n",
       "concepts/routes",
       "concepts/webviews",
-      "concepts/nlu",
-      "concepts/webchat"
+      "concepts/nlu"
     ],
     "Custom Webchat": [
-      "webchat/webchat-api",
+      "webchat/webchat",
+      "webchat/webchat-compatibility",
       "webchat/webchat-enable",
       "webchat/webchat-style",
       "webchat/webchat-covercomponent",
       "webchat/webchat-persistentmenu",
       "webchat/webchat-markdown",
       "webchat/webchat-blockinput",
       "webchat/webchat-timestamp",
-      "webchat/webchat-updates"
+      "webchat/webchat-updates",
+      "webchat/webchat-api",
+      "webchat/webchat-storage"
```

Comment:
notice if you need to add any redirect in `.redirects` when changing/adding new sections